packages/camera/camera_android/android/src/main/java/io/flutter/plugins/camera/features/exposurepoint/ExposurePointFeature.java

```diff
@@ -24,6 +24,7 @@ public class ExposurePointFeature extends CameraFeature<Point> {
   @Nullable private Point exposurePoint;
   private MeteringRectangle exposureRectangle;
   @NonNull private final SensorOrientationFeature sensorOrientationFeature;
+  private boolean defaultRectangleHasBeenSet = false;
```

Comment:
This should accompanied with another variable that stores the default rectangle. For example,

```java
@Nullable
private MeteringRectangle[] defaultRegions;
```

This should be set in the `updateBuilder` method right after checking that the feature is supported.

Also, this variable could probably be called `defaultRegionsHasBeenSet`. I made a mistake and forgot this is a list.